assets/js/atomic/utils/get-layout-config.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Converts inner blocks to a list of layout configs.
+ *
+ * @param {Object[]} innerBlocks Inner block components.
+ */
+export const getLayoutConfig = ( innerBlocks ) => {
```

Comment:
Moved from base-utils. This might be removable in a follow up when layout configs are deprecated/removed.